Formula/h/harpwise.rb

```diff
@@ -0,0 +1,33 @@
+require "fileutils"
+
+class Harpwise < Formula
+  desc "Harmonica practice-utility for the command-line using microphone and speaker"
+  homepage "https://github.com/marcIhm/harpwise"
+  url "https://github.com/marcihm/harpwise/archive/4.4.4.tar.gz"
+  sha256 "7caae0c4732bb188d164d86a4cceb2aa7b4f09859b945bc86f42d31500571e83"
+  license "MIT"
+
+  # dependencies ordered according to "brew style harpwise"
+  depends_on "aubio"
+  depends_on "coreutils"
+  depends_on "figlet"
+  depends_on "ruby"
```

Comment:
does it work with old ruby?

```suggestion
  uses_from_macos "ruby"
```

Reply:
Probably not with ruby 2.6; added a new commit to revert to ruby from homebrew. Thanx for the hint !

Reply:
yeah, for os with old ruby, you can do special syntax `uses_from_macos "ruby", since: :monterey`